### PR TITLE
Import some original drama design

### PR DESCRIPTION
**Pull request by duarte**
Imports some of the design tweaks originally made for original dramas

### CODE REVIEW

**Comment by duarte** [comment]
This PR is all smaller CSS changes, just keeping the master branch in sync…

File: src/css/info.scss [hunk 1]
```diff
@@ -77,6 +68,15 @@
         .expanded + .intro__expand {
             @include icon-showless;
         }
+        a {
+            color: inherit;
+            border-bottom: 1px solid $c-divider;
+            text-decoration: none;
+            &:hover {
+                color: #f6f6f6;
```
Comment (akash1810) [inline src/css/info.scss]: variable?

File: src/css/_general.scss [hunk 2]
```diff
@@ -81,3 +81,7 @@ a {
         transform: translateX(calc(100%));
     }
 }
+
+html {
```
Comment (akash1810) [inline src/css/_general.scss]: hmm, why is this needed?

Comment (duarte) [inline src/css/_general.scss]: Do you mean the `calc(100%)`, or the `html { min-height… }`

Comment (akash1810) [inline src/css/_general.scss]: the `html` rule

**Comment by duarte** [comment]
@akash1810 @gids — is this ok to merge?
